<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tankGUIClass</class>
 <widget class="QMainWindow" name="tankGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tankGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Extracted color : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="getColorButton">
       <property name="text">
        <string>getColor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="extractedColorLabel">
       <property name="text">
        <string>color here</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>531</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>camera image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="contoursLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>501</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>contours of maze</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>710</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image from camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>710</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>extracted lines</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Komunikacja z robotem</string>
    </property>
   </widget>
   <widget class="QLabel" name="tankReadyLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuinfo">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actioninfo"/>
   </widget>
   <addaction name="menuinfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioninfo">
   <property name="text">
    <string>info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tankGUI.qrc"/>
 </resources>
 <connections/>
</ui>
